docs/Update-SSHKeys.md

```diff
@@ -42,85 +58,100 @@ spec:
+```sh
+oc get machineconfigs 99-worker-ssh -oyaml --export > update-ssh-worker.yaml
+```
 
- - Updating `user: name`: Do not update the `user: name` field. The only user currently supported is `core` as shown in the above example config.
+```yaml
+# update-ssh-worker.yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-worker-ssh
+spec:
+  config:
+    passwd:
+      users:
+      - name: core
+        sshAuthorizedKeys:
+        - ssh rsa ABC123....
+        - ssh rsa XYZ7890....
+```
 
- ## Applying the MachineConfig
+Now with your MachineConfig yaml file (using the example above):
 
-Now with your new MachineConfig yaml file (using the example above):
 ```sh
-    oc create -f example-ssh-update.yaml
+oc apply -f update-ssh-worker.yaml
 ```
 
-You should see the new MachineConfig name appear almost immediately, from our example config:
-```sh
-    oc get machineconfigs
-
-    NAME                                                        GENERATEDBYCONTROLLER              IGNITIONVERSION   CREATED
-    00-master                                                   4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             87m
-    00-master-ssh                                               4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             87m
-    00-worker                                                   4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             87m
-    00-worker-ssh                                               4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             87m
-    01-master-container-runtime                                 4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             87m
-    01-master-kubelet                                           4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             87m
-    01-worker-container-runtime                                 4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             87m
-    01-worker-kubelet                                           4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             87m
-    99-master-9c65f9fb-41d0-11e9-994d-02360d172130-registries   4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             84m
-    99-worker-9c67b221-41d0-11e9-994d-02360d172130-registries   4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             84m
-    rendered-master-a1884339a91f02f19898fe9c5929256b            4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             87m
-    rendered-worker-777a036887bb25188f47c6d3219eabb9            4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             2s
-    rendered-worker-e99b9948bccfd387c71397b31e2c985e            4.0.0-alpha.0-83-g0a84145f-dirty   2.2.0             87m
-    ssh-workers-01                                                                                                   7s
+You should see the new MachineConfig name appear immediately, from our example config:
+
```

Comment:
This output is out of date, update it so it is correct, see PR #533

Reply:
updated.